thriftpy/parser/parser.py

```diff
@@ -11,478 +11,80 @@
 import os
 import sys
 import types
-from ply import lex, yacc
-from .lexer import *  # noqa
+import parsley
 from .exc import ThriftParserError, ThriftGrammerError
```

Comment:
wish this were `ThriftGrammarError` instead of `ThriftGrammerError`. Maybe I can make a separate PR to alias that :P